### PR TITLE
week6_이다현_복습과제

### PR DESCRIPTION
**Pull request by hopebii**

### CODE REVIEW

**Comment by greenmoonhg** [comment]
다현님 안녕하세요!
우선 과제 피드백이 늦어져 죄송합니댜 ㅠㅅㅠ

1.5 
이미 알고 계신 것 같지만!
1.5 문제의 해당 리스트에 어떤 unique한 값이 있는 지 확인할 때는 
unique()함수를 이용하시면 더 편리할 것 같아요!
-> ex. HR['Attribution].unique

1.6
직관적으로 더 잘 변환하신 코드같아요!
참고로
HR['Attrition'] = HR.Attrition.astype("category").cat.codes
위 코드로 간단하게 해결 가능하다는 점도 봐두시면 좋을 것 같아요!

1.8 
label encoding 을 할 때 조금 더 도움이 되실 것 같은 for 문을 첨부해드려요!

for column in ['BusinessTravel','Department','EducationField','Gender','JobRole','MaritalStatus','OverTime']:
    X[column] = label.fit_transform(X[column])

1.13, 1.14, 1.15
예측의 정확도는 X_test을 생성한 모델에 넣으면 나오는 값과 y_test를 비교해야 합니다. 
해당 코드에서 모델에 pred=모델.predict(X_test) 이 빠진 것 같아요!
꼭 확인 부탁드립니다!

수고하셨습니다 XD!!
